demo/api/schema.gql

```diff
@@ -525,6 +539,7 @@ input ProductFilter {
   title: StringFilter
   slug: StringFilter
   description: StringFilter
+  type: ProductTypeEnumFilter
```

Comment:
This does not look right as news are not touched by this PR

Reply:
News use the api generator and already used an enum. With this PR that enum is now supported and so code is generated.

In theory this shows that adding features to the generator can be incompatible. In pratice everyone uses it as scaffolding. Maybe we should declare api-generator as an unstable feature to allow such changes.

Reply:
yes, but then we don't have the feature in 4.

I mean, no one is using it as "real" generator, right?

but I can swtich to next if you prefer that

Reply:
I'd say we treat it as non-breaking. None of our projects is using the generator for _real_ generation atm.